src/components/DocumentManager/DocumentManager.cypress.js

```diff
@@ -72,16 +72,16 @@ describe("DocumentManager", function() {
     cy.visit(`?collection=${this.collectionId}`);
 
     // viewer state when DOM loaded but no loaded document
-    cy.get("[data-testid=document_viewer_no_documents]").should("exist");
+    cy.get("[aria-label='Document viewer with no document']").should("exist");
 
     // should appear when document loading
-    cy.get("[data-testid=spinner]").should("be.visible");
+    cy.get("[aria-label='Spinner icon']").should("be.visible");
```

Comment:
This is one of the rare instances in the PR where aria-label is justified. There's no other good way to query this, and adding the label helps a11y. 👍